<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapChunk_Settings</class>
 <widget class="QDialog" name="MapChunk_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Chunk index:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="chunkIndexLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="textureScaleTabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="textureScale0">
        <attribute name="title">
         <string>Base texture</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_ts0">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Texture scale options:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Texture scale far:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Texture scale near:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="textureScaleOptions0">
             <property name="currentText">
              <string>Both multiply</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Both multiply</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Both divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far multiply, Near divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far divide, Near multiply</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="textureScale1">
        <attribute name="title">
         <string>Layer 1</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_ts1">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Texture scale options:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="textureScaleOptions1">
             <property name="currentText">
              <string>Both multiply</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Both multiply</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Both divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far multiply, Near divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far divide, Near multiply</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Texture scale far:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Texture scale near:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Automatic texture:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Start height:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="automaticTexture1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>End height:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="startHeight1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32768.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDoubleSpinBox" name="endHeight1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32768.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="textureScale2">
        <attribute name="title">
         <string>Layer 2</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_ts2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Texture scale options:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="textureScaleOptions2">
             <property name="currentText">
              <string>Both multiply</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Both multiply</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Both divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far multiply, Near divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far divide, Near multiply</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Texture scale far:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Texture scale near:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Automatic texture:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Start height:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>End height:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDoubleSpinBox" name="endHeight2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32768.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="startHeight2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32768.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="automaticTexture2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="textureScale3">
        <attribute name="title">
         <string>Layer 3</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_ts3">
           <item row="0" column="1">
            <widget class="QComboBox" name="textureScaleOptions3">
             <property name="currentText">
              <string>Both multiply</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Both multiply</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Both divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far multiply, Near divide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Far divide, Near multiply</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Texture scale near:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Texture scale far:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Texture scale options:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Start height:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Automatic texture:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>End height:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="startHeight3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32768.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDoubleSpinBox" name="endHeight3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <double>-32768.000000000000000</double>
             </property>
             <property name="maximum">
              <double>32768.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="automaticTexture3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MapChunk_Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MapChunk_Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
